application_sandbox/write_timestamp/main.cpp

```diff
@@ -334,10 +333,10 @@ class WriteTimestampSample
     // Reset the query for this frame in the query pool and then begin query,
     // and so the vkGetQueryPoolResults called before submitting rendering
     // commands won't hang.
-    cmdBuffer->vkCmdPipelineBarrier(
-        cmdBuffer, VK_PIPELINE_STAGE_TRANSFER_BIT,
-        VK_PIPELINE_STAGE_COLOR_ATTACHMENT_OUTPUT_BIT, 0, 0, nullptr, 1,
-        &to_use_query_results, 0, nullptr);
+    cmdBuffer->vkCmdPipelineBarrier(cmdBuffer, VK_PIPELINE_STAGE_TRANSFER_BIT,
+                                    VK_PIPELINE_STAGE_2_FRAGMENT_SHADER_BIT_KHR,
```

Comment:
Could this be VK_PIPELINE_STAGE_FRAGMENT_SHADER_BIT, I think that would make it more "Vulkan 1.0" rather than using the vulkan 1.3 / synchronization2 flags.

Reply:
Yeah, I just missed this when back-porting the fixes from synchronization2_write_timestamp.